en/TOC.md

```diff
@@ -68,6 +68,9 @@
       - [Restore Data Using BR](restore-from-gcs-using-br.md)
       - [Back Up Data Using Dumpling](backup-to-gcs.md)
       - [Restore Data Using TiDB Lightning](restore-from-gcs.md)
+    - Azure Blob Storage
+      - [Back up Data Using BR](backup-to-azblob-using-br.md)
+      - [Restore Data Using BR](restore-from-azblob-using-br.md)
```

Comment:
Do we need to add Back up Data to Azure Blob Storage and Restore Data from Azure Blob Storage to backup-restore-overview.md?

Reply:
Added. Thanks!